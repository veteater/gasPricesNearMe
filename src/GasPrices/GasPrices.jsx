import React from 'react';

const GasPrices = () => {
  return (<div>Gas Prices Near Me</div>);
};

export default GasPrices;