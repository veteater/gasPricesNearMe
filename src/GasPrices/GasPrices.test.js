import React from 'react';
import { shallow } from 'enzyme';
import GasPrices from './GasPrices';

describe('GasPrices', () => {
  it('renders without crashing', () => {
    const component = shallow(<GasPrices />);
    expect(component).toMatchSnapshot();
  });
});
