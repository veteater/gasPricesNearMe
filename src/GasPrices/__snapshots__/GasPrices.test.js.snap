// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GasPrices renders without crashing 1`] = `
<div
  id="GasPrices"
>
  <Header />
  <Results />
</div>
`;
